### DISCUSSION TITLE
Search pop-up not announced as expanded

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3461] [Accessibility]
Priority level 1 (out of 3) from UW Accessible Technologies

WCAG mapping: 4.1.2 Name, Role, Value

STEPS TO REPRODUCE:
1. With a screen reader navigate to the search menu at the top of the page.
2. Open the submenu with enter/space and note how there is no announcement that the submenu has been expanded.

OBSERVED BEHAVIOR: 
Whenever opening the search submenu, there is no announcement that the submenu has been expanded. There is also inconsistent announcements for when the submenu is collapsed.

EXPECTED BEHAVIOR: 
There should be expanded/collapsed announcements whenever a menu is opened/closed.

**Comment by tsmyre** [comment]
**CIC Comment**
Isn’t `aria-expanded` adequate for this?

**Washington Comment**
With NVDA/Firefox the menu is announced as expanded, but not when it is collapsed. With all other combinations of NVDA/JAWS and Firefox/Chrome, there is no announcement that the menu is expanded or collapsed (unless you use the mouse to expand it for some reason). Having aria-expanded on the button is the correct practice so since that is being done I do not think there is anything else to be done and this may be a screen reader issue.
